sycl/test-e2e/KernelCompiler/kernel_compiler_sycl.cpp

```diff
@@ -8,11 +8,7 @@
 
 // REQUIRES: (opencl || level_zero)
 // UNSUPPORTED: accelerator
-
-// Flaky timeout on CPU. Enable when fixed.
-// Depends on SPIR-V Backend & run-time drivers version.
-// UNSUPPORTED: spirv-backend && cpu
-// UNSUPPORTED-TRACKER: CMPLRLLVM-64705
```

Comment:
Wasn't this recently added? Why is it safe to remove now?

If it's not, we might also want to consider whether the cache version fails flakily too.

Reply:
In the ticket they just have some that were timing out, not failing.  That's why I'm breaking this up.  The original test does  4 different calls to the kernel compiler for completeness, and then the cacheing stuff just multiplies that.  And that was creating needless time.  So I split the tests so we don't multiply everything by cache/no-cache/memcheck stuff.